src/timepicker/timepicker.ts

```diff
@@ -267,7 +268,7 @@ export class NgbTimepicker implements ControlValueAccessor,
     }
   }
 
-  formatMinSec(value: number) { return padNumber(value); }
+  formatMinSec(value?: number) { return padNumber(isNumber(value) ? value : NaN); }
```

Comment:
I don't how exactly how to manage this case, but I'm not comfortable with this code. The point is that I think that the isNumber is used too many times : `formatMinSec` checks `value` to send value or NaN, then padNumber checks `value` (isNumber again !) to send back the padded value or an empty string. 

Maybe one solution is to change padNumber in order to accept `value: any` (as isNumber) and remove the extra check ?

Reply:
Agreed

Reply:
Actually in this case I just aligned the implementation with the `formatHour` function, so I think I'll keep it as is